docs/proposal-multi-user-approach.md

```diff
@@ -44,71 +44,68 @@ The config discussed at [CylcCon2020](https://github.com/cylc/cylc-uiserver/issu
 
 ### READ and ADMIN
 
-For the purposes of clarity for users, and since we have operation (mutation) level granularity, the simpler method of having two groups, `READ` and `ADMIN`, rather than three. Access groups should be entered in caps to avoid confusing e.g. READ with read mutation.
+For the purposes of clarity for users, and since we have operation (mutation) level granularity, the simpler method of having two groups, `READ` and `ADMIN`, rather than three. These two access groups can be combined for ease of configuration with `ALL`. Access groups should be entered in caps to avoid confusing e.g. READ with read mutation.
```

Comment:
Thinking back to previous discussions, I think we wanted to put "broadcast" into an "edit" category?

(note tha Cylc8 trigger-edit functionality will be a combination of broadcast+trigger)

Reply:
Are you thinking to have `READ`, `CONTROL` and `EDIT`?
Where `EDIT` has both `broadcast` and `trigger` in it?

Reply:
`EDIT` would just have `broadcast` in it (because broadcast allows us to change what the workflow runs), `trigger` is a regular `CONTROL` cause all it does is run what is defined in the `flow.cylc` file. `trigger-edit` is a Cylc 7 thing that we are replacing with a combination of broadcast + trigger.

Reply:
Not sure it's worth having `EDIT` if it only has one thing (can always add it later).
If we want `CONTROL` to only allow users to interact with existing workflows then I think we also ought to exclude `install` and `re-install` (also `edit` and `terminal access` although I'm not sure we'll be able to provide multi-user support for these anyway). `reload` and `play` also have the potential to change the intended workflow behaviour but I think we have to accept that.

Reply:
I think it's worth it if it makes the corruptive nature of broadcast more apparent to sites which have to explicitly opt in to allowing users to give away that permission so can't be caught unaware. Doesn't matter that it's the only mutation in this group, I'm sure we can come up with more later!

Uninstalled workflows are still "existing workflows" so I'm not sure that there's a good reason to move them out of "control". You can only provide install options that are supported by the workflow anyway. Same for reload and play.

Reply:
(sidenote: from a security perspective broadcast and full terminal access are no different, they both allow unchecked arbitrary code execution so if we are going to allow one there shouldn't be a barrier to allowing the other).